planning/planning_error_monitor/config/planning_error_monitor.param.yaml

```diff
@@ -0,0 +1,7 @@
+/**:
+  ros__parameters:
+    # trajectory check
+    error_interval: 100.0 # error interval distance threshold [m]
+    error_curvature: 2.0 # error curvature threshold
+    error_sharp_angle: 0.785398 # error sharp angle threshold
+    ignore_too_close_points: 0.01 # ignore too close distance threshold
```

Comment:
In the same way as `error_interval`, adding units to all parameters would be better.

Reply:
After check some units, I will add them in yaml file.